### ISSUE TITLE
How is it supposed to work

### ISSUE THREAD

**Issue by steinitz-zz** [xcv58/meteor-chess#1]
Hi,

Your app is very interesting.  However, I can't understand how its supposed to work.  What generates the tokens etc?

In the mean time I've commented out the token etc code and the board seems to work, although an extra piece temporarily appears when I drag-move a piece.

Anyway, great work and I hope my comments are useful.

Cheers,

Steve

**Comment by xcv58** [comment]
I'm sorry for misleading you.

This is not a regular chess game. It aims to provide a cooperative chess game. I mean, one player with one token can only play one step.

You need to change token related codes, if you wanna a regular chess game.

Hope this can solve your problem.

Thanks,
xcv58

**Comment by steinitz-zz** [comment]
Thanks for your friendly reply.  I don't feel misled :)

Ah I see, a cooperative game.  Cool.

I've modified it to work as a normal game.  What I was really asking is how it's supposed to work.  I'd like to try it, but can't see how to make it work.  Maybe it's not ready yet?  No rush on my part.

Anyway, please, no need to apologize - no problems here.  I'm just curious.  It looks interesting. Nice piece of work.

Cheers,

Steve

**Comment by xcv58** [comment]
It actually uses http://chessboardjs.com to drive chess game.
So you can find full document from http://chessboardjs.com/docs

Thanks,
xcv58

**Comment by steinitz-zz** [comment]
Yes, I'm familiar with the chessboardjs and chessjs libraries.  I was more asking about your code, how the tokens etc are supposed to work.  Out of the box, it's a head scratcher :)

Cheers,

Steve

**Comment by xcv58** [comment]
It's my bad. I only use this as course project, focus on the cooperation. The tokens are for participants, to make sure each participant can only play one step.

Best,
xcv58
